CNAME

```diff
@@ -0,0 +1 @@
+samples.avenard.org
```

Comment:
Not sure why this is necessary.

Reply:
It looks like it's created automatically by GitHub as I configured it to serve content with name samples.avenard.org it wasn't in my original push. I'll create another branch